src/footer/selection/selection.ts

```diff
@@ -179,7 +179,11 @@ export class FooterSelection extends DefaultControl {
       $(this.map._container).find('.mapboxgl-ctrl-bottom-right').css('bottom', selected_height)
     }
 
-    if (isFunction(this._options.shouldShowInformationButtonFor) && this._options.shouldShowInformationButtonFor(element)) {
+    const informationButton = this._options.shouldShowInformationButtonFor
+
+    if (isFunction(informationButton) && isString(informationButton(element)) && !isEmpty(informationButton(element))) {
+      this._container.find('#mwz-footer-informations-button').replaceWith(informationButton(element))
```

Comment:
```suggestion
      this._container.find('#mwz-footer-informations-button').html(informationButton(element))
```
Don't replace the entire element, because it will work only once